buildutil/tap-test

```diff
@@ -13,7 +13,7 @@ touch ${tempdir}/.testtmp
 function cleanup () {
     if test -n "${TEST_SKIP_CLEANUP:-}"; then
 	echo "Skipping cleanup of ${tempdir}"
-    else if test -f ${tempdir}/.test; then
+    else if test -f ${tempdir}/.testtmp; then
```

Comment:
This mainly affects `gnome-desktop-testing-runner`, which is how I run the installed-tests (but not the build-time tests) for Debian.